File: biblioteca/src/main/java/com/jambodev/biblioteca/entities/Livro.java
package com.jambodev.biblioteca.entities;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Data;

@Entity @Data
public class Livro {

    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    
    private String titulo;
    private String autor;
    private String capa;
    private boolean disponivel;

}

File: biblioteca-front/src/app/models/livro.ts
export class Livro{
    titulo: string;
    autor: string;
    ano: number;
    capa: string;
    disponibilidade: boolean;
  
    constructor(titulo: string, autor: string, ano: number, capa: string, disponibilidade: boolean){
      this.titulo = titulo;
      this.autor = autor;
      this.ano = ano;
      this.capa = capa;
      this.disponibilidade = disponibilidade;
    }
  }